tests/test_stereoshift_dialog.py

```diff
@@ -90,8 +96,6 @@ def test_calculate_stereoshift_ui(
             test_points[i + 2] - test_points[i % 2]
         )
 
-    assert dlg.cbf1.currentIndex() == 0
```

Comment:
?

I also don't know what `dlg.cbf1` means. We could do with a more readable naming convention.
`dlg` is the stereoshift dialog that we're testing? So `steroshift_dialog.<something>`?

Reply:
This is the combobox for fiducial 1, but agreed, it is a bit cryptic. @Joseph-Garvey is working on the UI for the stereoshift dialog (#112), which includes some renaming of the buttons.